.github/workflows/validate.yml

```diff
@@ -37,6 +37,30 @@ jobs:
         with:
           targets: ${{ matrix.target }}
 
+  # check that the generated Markdown and the checked-in files match
+  validate-md:
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
nit: don't think this is needed (will speed up fetch time)

Reply:
LOL, bit confused by your comment; Will _removing_ it improve fetch time or _keeping_ it will? (I assumed "0" normally would safe time, but perhaps with caching / different checks doing without this, things may be different?)

Reply:
Default is `1`: https://github.com/actions/checkout/blob/8f4b7f84864484a7bf31766abe9204da3cbe65b3/action.yml#L56-L58

> 0 indicates all history for all branches and tags.

Reply:
Oh! Good catch. I also would've assumed `0` for "shallow", not the reverse 😬 😂

Reply:
fixed, PTAL